test/unit/player.test.js

```diff
@@ -363,9 +364,13 @@ QUnit.test('should suppress source error messages', function(assert) {
   assert.strictEqual(errors, 1, 'error after click');
 
   player.dispose();
+
+  assert.strictEqual(log.error.callCount, 2, 'two stubbed errors');
```

Comment:
`foo` is not a tech so of course we throw errors. That is what the test is testing anyway.